lib/internal/bootstrap_node.js

```diff
@@ -54,6 +54,8 @@
 
     _process.setupRawDebug();
 
+    NativeModule.require('internal/url');
```

Comment:
I'm a bit afraid of the increase in memory usage. Is there a need to make some requires in `internal/url` lazy?